src/sentry/tasks/post_process.py

```diff
@@ -24,10 +24,17 @@
 
 logger = logging.getLogger("sentry")
 
-
 locks = LockManager(build_instance_from_options(settings.SENTRY_POST_PROCESS_LOCKS_BACKEND_OPTIONS))
 
 
+class PostProcessJob(TypedDict, total=False):
+    event: "Event"
+    group_state: Dict[str, Any]  # TODO wait for barkshark typed dict
```

Comment:
waiting for https://github.com/getsentry/sentry/commit/b5fa9faa20b76697ab7cf681b2aa4725cf3dc5ac to be reverted